### PR TITLE
Update ChangeLog for FreeRTOS+TCP version and security improvement text

### PR DESCRIPTION
**Pull request by aggarw13**
Update **FreeRTOS+TCP** section in ChangeLog for **202007.00** release

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -55,13 +55,17 @@
 - Made fixes to IotBle_On() and IotBle_Off() APIs.
 - Accommodate larger-than-expected writes to RXLargeMesg Gatt Characteristic.
 
-#### FreeRTOS+TCP V2.2.0
+#### FreeRTOS+TCP V2.3.0
 
 - Added ability to cache multiple IP addresses per DNS entry.
+- Defensive security improvements: 
+    - In compliance with the UDP protocol specification, prior versions of FreeRTOS+TCP accepted UDP packets that had their checksum set to 0. FreeRTOS+TCP V2.3.0 adds a new configuration parameter, `ipconfigUDP_PASS_ZERO_CHECKSUM_PACKETS`, that enables users to opt to drop packets UDP packets that have their checksum set to 0. **Note:** This new setting defaults to 0, so it defaults to dropping UDP packets that have their checksum set to 0.
```
Comment (RichardBarry) [inline CHANGELOG.md:62]: There is a typo "to drop  packets UDP packets".

Comment (aggarw13) [reply CHANGELOG.md]: Good catch. Fixed

**Pull request merged**